app/src/main/java/de/danoeh/antennapod/fragment/preferences/StatisticsFragment.java

```diff
@@ -101,11 +122,43 @@ private void selectStatisticsMode() {
             listAdapter.setCountAll(countAll);
             prefs.edit().putBoolean(PREF_COUNT_ALL, countAll).apply();
             refreshStatistics();
+            getActivity().invalidateOptionsMenu();
         });
 
         builder.show();
     }
 
+    private void confirmResetStatistics() {
+        if (!countAll) {
+            ConfirmationDialog conDialog = new ConfirmationDialog(
+                    getActivity(),
+                    R.string.statistics_reset_data_title,
+                    R.string.statistics_reset_data_msg) {
+
+                @Override
+                public void onConfirmButtonPressed(
+                        DialogInterface dialog) {
```

Comment:
Please do not break this line

Reply:
Oups sorry, I copy pasted this block from `FeedItemlistFragment` and didn't think this was not correct.